src/vivarium_testing_utils/automated_validation/data_loader.py

```diff
@@ -163,7 +167,13 @@ def _load_from_artifact(self, data_key: str) -> Any:
         return data
 
     def _load_from_gbd(self, data_key: str) -> Any:
-        data = load_standard_data(data_key, self.location)
+        if "categories" in data_key:
```

Comment:
This update is to make loading GBD compatible with the Exposure, RelativeRisk, and PopulationStructure RatioMeasure classes.